templates/_header.html

```diff
@@ -22,11 +22,30 @@
         </li>
         <li class="p-navigation__link" role="menuitem"><a href="https://build.snapcraft.io">Build</a></li>
         <li class="p-navigation__link" role="menuitem">
-          <a class="p-link--external{% if page_slug and page_slug == 'my-snaps' %} is-selected{% endif %}" href="https://dashboard.snapcraft.io/snaps">My snaps</a>
+          <a class="{% if page_slug and page_slug == 'my-snaps' %} is-selected{% endif %}" href="/account/snaps">My snaps</a>
```

Comment:
if we comment out the login/ logout we should also add this in a comment and replace it with the 'external-link' to dashboard